start.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+set -eu
```

Comment:
Adding `x` option helps when commands fail so you know which command was being run.

Reply:
Yeah, agreed